client/app/src/main/java/com/example/mrides/controller/RequestHandler.java

```diff
@@ -170,7 +170,7 @@ public void Notify(String response) {
 
         for(ActivityObserver e : observers){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.